documentation/docs/13-configuration.md

```diff
@@ -72,6 +73,18 @@ An object containing zero or more of the following `string` values:
 - `hooks` — the location of your hooks module (see [Hooks](#hooks))
 - `template` — the location of the template for HTML responses
 
+### floc
+
+Google's [FLoC](https://github.com/WICG/floc) is a technology for targeted advertising that the [Electronic Frontier Foundation](https://www.eff.org/) has deemed [harmful](https://www.eff.org/deeplinks/2021/03/googles-floc-terrible-idea) to user privacy. [Browsers other than Chrome](https://www.theverge.com/2021/4/16/22387492/google-floc-ad-tech-privacy-browsers-brave-vivaldi-edge-mozilla-chrome-safari) have declined to implement it.
+
+In common with services like [GitHub Pages](https://github.blog/changelog/2021-04-27-github-pages-permissions-policy-interest-cohort-header-added-to-all-pages-sites/), SvelteKit protects your users by automatically opting out of FLoC. It adds the following header to responses unless `floc` is `true`:
+
+```
```

Comment:
Does the current implementation of syntax highlighting correctly handle syntax-less code blocks? I recall that at some point this was an issue on the Svelte site.

Reply:
this suggests it's optional: https://github.com/sveltejs/action-deploy-docs/blob/42d39f22a94353de238f9e2382645458e1ebb99f/src/format/code.ts#L29-L30

Reply:
Ah! Okay. I was poking around for where that code lived now but wasn't finding it.